src/System.DirectoryServices.AccountManagement/Directory.Build.props

```diff
@@ -1,4 +1,4 @@
-﻿<Project>
+<Project>
```

Comment:
Could you revert this changes? It seems like your code is changing characters in our files.

Reply:
Probably adding or removing the BOM.